core/src/components/datetime/utils/format.ts

```diff
@@ -11,7 +11,33 @@ const getFormattedDayPeriod = (dayPeriod?: string) => {
   return dayPeriod.toUpperCase();
 };
 
-export const getLocalizedTime = (locale: string, refParts: DatetimeParts, hourCycle: DatetimeHourCycle): string => {
+/**
+ * Including time zone options may lead to the rendered text showing a
+ * different time from what was selected in the Datetime, which could cause
+ * confusion.
+ */
+export const stripTimeZone = (formatOptions: Intl.DateTimeFormatOptions): Intl.DateTimeFormatOptions => {
+  /**
+   * We do not want to display the time zone name
+   */
+  delete formatOptions.timeZoneName;
```

Comment:
I do not think we should be using `delete` here. 

This will mutate the original reference passed into the function. 

For example, check the console logs in this example: https://codepen.io/sean-perkins/pen/BabMMBK.

You will see that after, the object passed into the helper function no longer reflects it's original value.

Instead, we should be creating a new object _without_ the timezone.

```ts
return {
  ...formatOptions
  timeZone: 'UTC',
  timeZoneName: undefined
}
```

Reply:
fixed in d9239e1b56